docs/API-REFERENCE.md

```diff
@@ -1 +1,2640 @@
+    ...
+  }
+
+  jerry_release_value (value);
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_value_is_object
+
+**Summary**
+
+Returns whether the given `jerry_value_t` is an object.
+
+**Prototype**
+
+```c
+bool
+jerry_value_is_object (const jerry_value_t value);
+```
+
+- `value` - api value
+- return value
+  - true, if the given `jerry_value_t` is an object value
+  - false, otherwise
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_value_t value;
+  ... // create or acquire value
+
+  if (jerry_value_is_object (value))
+  {
+    ...
+  }
+
+  jerry_release_value (value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_value_is_string
+
+**Summary**
+
+Returns whether the given `jerry_value_t` is a string.
+
+**Prototype**
+
+```c
+bool
+jerry_value_is_string (const jerry_value_t value);
+```
+
+- `value` - api value
+- return value
+  - true, if the given `jerry_value_t` is a string
+  - false, otherwise
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_value_t value;
+  ... // create or acquire value
+
+  if (jerry_value_is_string (value))
+  {
+    ...
+  }
+
+  jerry_release_value (value);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+
+
+# jerry_acquire_value
+
+**Summary**
+
+Acquires the specified Jerry API value.
+
+For values of string, number and object types this acquires the underlying data, for all other types
+it is a no-op. The acquired pointer have to be released by [jerry_release_value](#jerry_release_value).
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_acquire_value (jerry_value_t value);
+```
+
+- `value` - api value
+- return value - pointer that may be used outside of the engine
+
+**Example**
+
+```c
+{
+  jerry_init (JERRY_FLAG_EMPTY);
+  jerry_object_t *object_p = jerry_create_object ();
+  jerry_value_t object_value = jerry_create_object_value (object_p);
+
+  jerry_value_t acquired_object = jerry_acquire_value (object_value);
+
+  jerry_release_value (acquired_object);
+  jerry_release_object (object_p);
+}
+```
+
+**See also**
+
+- [jerry_release_value](#jerry_release_value)
+- [jerry_acquire_string](#jerry_acquire_string)
+- [jerry_acquire_object](#jerry_acquire_object)
+
+
+# jerry_release_value
+
+**Summary**
+
+Release specified pointer to the value.
+
+**Prototype**
+
+```c
+void
+jerry_release_value (jerry_value_t value);
+```
+
+- `value` - api value
+
+**Example**
+
+```c
+{
+    jerry_value_t val1 = jerry_create_object_value (jerry_create_object ());
+
+    jerry_string_t *str_p = jerry_create_string ("abc");
+    jerry_value_t val2 = jerry_create_string_value (str_p);
+
+    ... // usage of val1
+
+    jerry_release_value (val1);
+
+    ... // usage of val2
+
+    jerry_release_value (val2);
+}
+```
+
+
+# jerry_create_string
+
+**Summary**
+
+Create new JavaScript string.
+
+Upon the JavaScript string becomes unused, all pointers to it should be released using
```

Comment:
`Upon` feels quirky. Can we say `When the string is no longer needed ...`.
Also, I wouldn't say here that `all pointers should be released`, only that `it has to be released`, as it is a bit misleading. All _acquired_ pointers should be released, but `jerry_acquire_string` already notes this, and that's fine, as it's only relevant there.